test/integration/node-specific/examples/aws_handler.js

```diff
@@ -0,0 +1,26 @@
+// begin lambda connection
+const { MongoClient } = require('mongodb');
+
+// Get the URI for the cluster then set AWS_ACCESS_KEY_ID as the username in the
+// URI and AWS_SECRET_ACCESS_KEY as the password, then set the appropriate
+// options.
+const connectionString = new URL(process.env.MONGODB_URI);
```

Comment:
This will crash for comma separated hostnames, if this is the recommended programatic way to build in these settings should we demonstrate that users can import ConnectionString from `'mongodb-connection-string-url'`? We could re-export it from the driver or users can rely on it always being present since its a driver dep. 

Or maybe we just demonstrate using template strings here to build the uri 🤷 I'm not sure which is better

Reply:
I just went with this way because it is the way the docs currently explain this, and the ticket was to recreate the docs. I personally agree there are better ways but maybe the docs then need to change as well?

Reply:
discussed offline: let's go with the programmatic way to pass options, and file a docs ticket to fixup the example (or is this going to replace that example anyway?)

Reply:
Updated the example to pass in the env variables as options to the client.